src/commands/create_command.py

```diff
@@ -16,8 +16,8 @@ def __init__(self, schema_file):
     def execute(self):
         os.makedirs(self.path, exist_ok=True)
         for table in self.data[Keys.TABLES]:
-            t_path = os.path.join(self.path, table[Keys.NAME])
-            os.makedirs(t_path, exist_ok=True)
+            table = TableMetaData(table)
+            table.serialize(self.path)
```

Comment:
As serialize takes self.path ? Do you mean it is database_path ? 
If yes, we don't pass database_path in the constructor